src/battle_script_commands.c

```diff
@@ -2334,38 +2334,26 @@ static inline bool32 DoesBattlerNegateDamage(u32 battler)
     return FALSE;
 }
 
-static u32 UpdateEffectivenessResultFlagsForDoubleSpreadMoves(u32 resultFlags)
+static u32 UpdateEffectivenessResultFlagsForDoubleSpreadMoves(u32 resultFlags, u32 moveTarget)
 {
-    // Only play the "best" sound
-    for (u32 sound = 0; sound < 3; sound++)
+    u32 battlerDef;
+    u32 ret = resultFlags;
```

Comment:
```suggestion
    u32 resultFlagsForSoundEffectiveness = resultFlags;
```
Doesn't have to be this verbose but maybe something more expressive? It's not clear from `ret` what we are returning.